<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionDialog</class>
 <widget class="QDialog" name="OptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>387</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OptionDialog</string>
  </property>
  <widget class="QComboBox" name="LogModeBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>170</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>0</height>
    </size>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>Advanced</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Advanced (No Log File)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Simple</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Simple (No Log File)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="LogModeLbl">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Logging mode:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DebugEnableLbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enable Debug Options</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="DebugEnableCheckBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>35</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TooltipLbl">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tooltip Color:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="TooltipBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>White</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Red</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="SaveBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>52</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="CancelBtn">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="UiModeLbl">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UI Mode:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="UiModeBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>This selects the UI design. Designs are taken from ./assets/ui</string>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="ReloadUIsBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </widget>
  <widget class="QLabel" name="TooltipLbl_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Section Name File:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="SectionFileLine">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenFileBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>OptionDialog</receiver>
   <slot>CancelBtn_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveBtn</sender>
   <signal>clicked()</signal>
   <receiver>OptionDialog</receiver>
   <slot>SaveBtn_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DebugEnableCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>OptionDialog</receiver>
   <slot>DebugEnableCheckState_Changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LogModeBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>OptionDialog</receiver>
   <slot>LoggingMode_IndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TooltipBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>OptionDialog</receiver>
   <slot>TooltipColor_IndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UiModeBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>OptionDialog</receiver>
   <slot>UiModeBox_IndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ReloadUIsBtn</sender>
   <signal>clicked()</signal>
   <receiver>OptionDialog</receiver>
   <slot>ReloadUIsBtn_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpenFileBtn</sender>
   <signal>clicked()</signal>
   <receiver>OptionDialog</receiver>
   <slot>OpenFileBtn_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>DebugEnableCheckState_Changed(int)</slot>
  <slot>LoggingMode_IndexChanged(int)</slot>
  <slot>TooltipColor_IndexChanged(int)</slot>
  <slot>SaveBtn_Clicked()</slot>
  <slot>CancelBtn_Clicked()</slot>
  <slot>UiModeBox_IndexChanged(int)</slot>
  <slot>ReloadUIsBtn_Clicked()</slot>
  <slot>OpenFileBtn_Clicked()</slot>
  <slot>SectionFileLine_TextChanged(QString)</slot>
 </slots>
</ui>
